xcm-bridge/src/lib.rs

```diff
@@ -0,0 +1,450 @@
+// The Licensed Work is (c) 2022 Sygma
```

Comment:
AFAIK, You can reuse orml/xtoken pallet implementation

Reply:
like we discussed, for now, we want to impl our own version, but use existing xtoken as a lib is always an option in the future for sure